contrib/grimshot.1.scd

```diff
@@ -23,6 +23,10 @@ grimshot - a helper for screenshots within sway
 *copy*
 	Copy the screenshot data (as image/png) into the clipboard.
 
+*save-copy*
+	Save the screeshot into a regular file (see above for save) and 
```

Comment:
- s/screeshot/screenshot
- mark save as `*save*` and copy as `*copy_*, since they refer to commands

Reply:
- misspelling corrected
- marked *save* and *copy* => hope my understanding was correct